### PR TITLE
add profile and CI step

### LINKED ISSUE #133: Databricks support
### Describe the feature
Allow databricks users to run the package

### Additional context
[Community has requested this functionality!](https://getdbt.slack.com/archives/C01NH3F2E05/p1655139152074579?thread_ts=1655126255.889019&cid=C01NH3F2E05)

### Who will this benefit?
Databricks users

### PR DESCRIPTION
**Pull request by dave-connors-3**
This is a:
- [ ] bug fix PR with no breaking changes
- [x] new functionality

## Link to Issue 
Closes #133 (techincally @b-per 's did but this makes it real!)

## Description & motivation
We should test databricks connections in our circle CI flow!

Check out #140 if you'd like to see the checks in action!

## Integration Test Screenshot
image

## Checklist
- [x] I have verified that these changes work locally on the following warehouses (Note: it's okay if you do not have access to all warehouses, this helps us understand what has been covered)
    - [ ] BigQuery
    - [ ] Postgres
    - [ ] Redshift
    - [ ] Snowflake
- [ ] I have updated the README.md (if applicable)
- [ ] I have added tests & descriptions to my models (and macros if applicable)
- [ ] I have added an entry to CHANGELOG.md

### CODE REVIEW

**Comment by b-per** [comment]
I just realised that giving a `name` to a test is only working in `1.1+`, so we will need to mention that the package only works for `>=1.1` or we can remove the name for the different tests. to make it compatible for `>=1.0`

**Comment by dave-connors-3** [comment]
> I just realised that giving a `name` to a test is only working in `1.1+`, so we will need to mention that the package only works for `>=1.1` or we can remove the name for the different tests. to make it compatible for `>=1.0`

I think I am ok with our forcing users up to 1.1! what do you think @b-per ? @graciegoheen ?

File: requirements.txt [hunk 1]
```diff
@@ -1,5 +1,6 @@
 
 pip==21.3.1
-dbt-snowflake~=1.0.0
-dbt-redshift~=1.0.0
-dbt-bigquery~=1.0.0
\ No newline at end of file
+dbt-snowflake~=1.1.0
+dbt-redshift~=1.1.0
+dbt-bigquery~=1.1.0
+dbt-databricks~=1.1.0
```
[the remark this answers is not in the document]
Comment (dave-connors-3) [reply requirements.txt:6]: thanks @dbeatty10 ! added this!

[the remark this answers is not in the document]
Comment (dbeatty10) [reply requirements.txt:6]: 🙌

**Pull request merged**